index.html

```diff
@@ -6,6 +6,7 @@
     <title>Les 4 - Opdracht transitie hamburger icoon naar kruisje</title>
 </head>
 <body>
-    <div id="menu"></div>
+<div class="container"><div id="menu"></div>
```

Comment:
hier heb je alleen het menu nodig

Reply:
<div id="menu"></div>

Reply:
In de uitwerking file index.html staat de container wel!